src/styles/card.scss

```diff
@@ -0,0 +1,115 @@
+.cards {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  box-sizing: border-box;
+  width: (4 * $card-width) + (3 * $card-margin-right) + (2 * $padding-horizont);
+  padding: $padding-vertical $padding-horizont;
+  margin: 0 auto;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 200px;
+  background-color: #{$white-color};
+  border: 1px solid #{$like-white-color};
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  margin-bottom: $card-margin-top;
```

Comment:
same external geometry. Use BEM mixes

Reply:
margin is still here
![image](https://user-images.githubusercontent.com/98764641/216319914-a117564e-d6b3-4fe8-8664-b8355366d7fd.png)